canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -883,6 +883,18 @@ Cypress.Commands.add("verifyLeftPanelWidth", (width) => {
 	});
 });
 
+Cypress.Commands.add("verifyLeftFlyoutPanelWidth", (width) => {
```

Comment:
I noticed the name of this function is very similar to the one directly above it. To avoid future confusion could you rename the one above to be `verifyPalettePanelWidth` and change all the places it is called from?